boards/arm/rpi_pico/rpi_pico-pinctrl.dtsi

```diff
@@ -15,4 +15,34 @@
 			input-enable;
 		};
 	};
+
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <I2C0_SDA_P4>;
+			bias-pull-up;
+			input-enable;
+			input-schmitt-enable;
+		};
+		group2 {
+			pinmux = <I2C0_SCL_P5>;
+			bias-pull-up;
+			input-enable;
+			input-schmitt-enable;
+		};
```

Comment:
```suggestion
		group1 {
			pinmux = <I2C0_SDA_P4>, <I2C0_SCL_P5>;
			bias-pull-up;
			input-enable;
			input-schmitt-enable;
		};
```

note: I2C resistor value depends on bus speed, so using internal pull-ups is, in general, a bad idea.

Reply:
Done!